Launch features

agnoster opened the pull request:
NOTE: This is a pull request after the fact to bring the version on Github in line with the one we're releasing.

slobak committed:
cp

slobak committed:
removed options

slobak committed:
moved success/error

slobak committed:
fix typekit

slobak committed:
styling

slobak committed:
changing layout, wip

slobak committed:
fixed API communication

slobak committed:
basic typeahead, no keyboard selection yet

slobak committed:
keyboard selection, kinda working

slobak committed:
better keyb selection

slobak committed:
working scrolling on keyboard

slobak committed:
Grab favicon url from page and fix some bugs

slobak committed:
Fix resize bug, and show contacts in order

slobak committed:
Fix task creation functionality, add logging

slobak committed:
Load correct typekit for locale

Commit:
Checkpoint.

Commit:
More changes.

Commit:
Finalized CSS.

slobak committed:
added logging, default assigned to logged-in user

slobak committed:
fix client name

slobak committed:
fixed initial rendering of assignee

slobak committed:
assignee looks like no one but is self by default

slobak committed:
cp

slobak committed:
Alt and Tab hotkeys are now more robust

slobak committed:
fill in field with selected user's name

slobak committed:
fix dimensions for popup

slobak committed:
select on hover

Commit:
Final styling.

slobak committed:
Updated comments

slobak committed:
Update signup link

slobak committed:
Log workspace change

slobak committed:
Increase photo resolution, better for retina display

slobak committed:
Fix breakage on workspace change when assignee already selected

Commit:
anonymous man

slobak committed:
Fix scrollwheel scrolling of assignee list

slobak committed:
Merge branch 'launch' of github.com:Asana/Chrome-Extension-Example into launch

slobak committed:
Added silhouette

slobak committed:
Update 128px icon

slobak committed:
Update manifest with more appropriate configuration

slobak committed:
Always put current user first

slobak committed:
Moved to use command API instead of a content script

slobak committed:
reduce content script execution to just getting selection

slobak committed:
Update README

Commit:
final fixes.

slobak committed:
Default assignee to current user, update filtered list when opened, reduce permissions by avoiding selection

slobak committed:
Don't focus popup body

slobak committed:
Use browser action for command and switch keyboard shortcut to Alt+Shift+A

slobak committed:
Fix wrapping on completion

slobak committed:
Fix broken font

slobak committed:
Update version for type fix

epelz commented on README:
```diff
@@ -4,11 +4,10 @@ integrates Asana into your web experience in the following ways:
+    the URL and current selected text in the notes.
 
-  * Installs the special Asana TAB+Q keyboard shortcut. When this key combo
+  * Installs the special Asana ALT+A keyboard shortcut. When this key combo
```
Isn't this actually `ALT+SHIFT+A`?

niieani replied:
If it is in fact ALT+A, that would be most unfortunate, as that key is used by certain international keyboards to write accented characters.

Commit:
Adding client header to API requests

Commit:
Fix banner icon alignment

Commit:
Update icons for visual relaunch (still does not have new styles in UI though)

chadworth committed:
Add a LICENSE file

All open source projects should describe the terms under which they are licensed. Most JavaScript projects like jquery, Angular, etc. are licensed under the MIT license, so that could be a good choice for this project.

slobak committed:
Merge pull request #10 from chadworth/patch-1

Add a LICENSE file

Commit:
Reduce tab permission to just what is necessary

periodic committed:
Changes the user-selection to use type-ahead instead of caching all the users in a domain.

periodic committed:
Removes unused user filtering code.

periodic committed:
Removes the XHR abort code since it doesn't actually do anything.

Commit:
Cleans up some debugging code and adds a missing semicolon.

Commit:
First pass at updated design: inputs, typeahead, logo, colors

Commit:
Further design tweaks - status messages, icons

Commit:
Bump manifest version

Commit:
Update font usage

Commit:
Remove typekit permissions and improve icon

slobak committed:
Merge pull request #12 from Asana/quick-redesign

Quick and dirty redesign for non-embarassing marketing

Commit:
Merges in UI updates.

Commit:
Minor fix-ups.

periodic committed:
Changes the user-selection to use type-ahead.

Changes the user-selection to use type-ahead.

Commit:
Fixes some style errors.

periodic committed:
Fixes some style errors.

Fixes some style errors that were missed in #11.

Commit:
Bumps manifest version.

periodic committed:
Merge pull request #15 from Asana/drewhaven-bump-manifest

Bumps manifest version to v 1.2.0

periodic committed:
Prevents duplicate click handlers on the button.

periodic committed:
Merge pull request #16 from Asana/drewhaven/too-many-tasks

Prevents duplicate click handlers on the button.
